src/arch/host/configs/library_defconfig

```diff
@@ -8,9 +8,11 @@ CONFIG_COMP_MFCC=y
 CONFIG_COMP_MODULE_ADAPTER=y
 CONFIG_COMP_MULTIBAND_DRC=y
 CONFIG_COMP_MUX=y
+#CONFIG_COMP_RTNR=y
```

Comment:
suggest directly delete line.

Reply:
Draft - I left this a reminder to fix an issue in testbench build. There's something that fails to compile. I'd want to be able load and run every component (also with mock) to testbench to do basic checks in CI (scripts/host-testbench.sh) to avoid regressions with more rare use.

If I can't solve the build issue I will drop all testbench related.